rosdep/base.yaml

```diff
@@ -3294,6 +3297,7 @@ nodejs-legacy:
 npm:
   debian: [npm]
   fedora: [npm]
+  gentoo: [net-libs/nodejs]
```

Comment:
AFAIK you can't `emerge net-libs/nodejs[npm]` (or something not involving the `USE` variable, e.g. some commandline switch for portage) and I don't know if someone want to change `rosdep` in order to support gentoo useflags.

@allenh1 TODO: compile a list of required package/use